lib/AppInfo/Application20.php

```diff
@@ -154,8 +155,8 @@ private function registerUserGroupHooks(): void {
 		});
 	}
 
-	public function registerCommentsEntity(): void {
-		$this->server->getEventDispatcher()->addListener(CommentsEntityEvent::EVENT_ENTITY, function (CommentsEntityEvent $event) {
+	public function registerCommentsEntity(SymfonyAdapter $symfonyAdapter): void {
```

Comment:
`\OCP\Comments\CommentsEntityEvent` is a "new" event based on `\OCP\EventDispatcher\Event`. Aren't you able to use our dispatcher abstraction?